contacts24/errors.py

```diff
@@ -1,6 +1,6 @@
 HELP_ERROR_MESSAGE = """Supported functions:
```

Comment:
You can dismiss this error message for now and undo changes, I made some slight updates in this [PR](https://github.com/konvio/python-contacts-cli-2024/pull/21) and will keep updating this as new functions are finished